presentation/src/main/java/com/anytypeio/anytype/presentation/widgets/TreeWidgetContainer.kt

```diff
@@ -31,6 +31,8 @@ class TreeWidgetContainer(
     isSessionActive: Flow<Boolean>
 ) : WidgetContainer {
 
+    private val rootLevelLimit = WidgetConfig.resolveTreeWidgetLimit(widget.limit)
```

Comment:
rootLevelLimit could be moved inside 'buildTree' function, I suppose?

Reply:
thanks, it's been there, but then i thought that each container is always association with the single widget, so this limit stays the same for the container, no need to calculate it each time